pages/index.js

```diff
@@ -17,7 +17,7 @@ export const getStaticProps = async () => {
   const date = new Date();
   const selectedImage = date.getDay();
   const image = heroImages[selectedImage];
-  const FORM_ID = process.env.FORM_ID;
```

Comment:
I think the best way to fix it right now is to add FORM_ID in `.env.example` with random string

Reply:
@akshitarora921 I have made the appropriate changes.
@LowLifeArcade Happy to help :)